examples/neural_compressor/language-modeling/run_clm.py

```diff
@@ -214,9 +226,24 @@ class OptimizationArguments:
     )
     quantization_methodology: str = field(
         default="RTN",
-        metadata={
-            "help": "Quantization methodology for weight only quantization. Choose from 'RTN', 'AWQ' and 'GPTQ'."
-        },
+        metadata={"help": "Quantization methodology for weight only quantization. Choose from 'RTN' and 'GPTQ'."},
+    )
+    gptq_percdamp: float = field(
+        default=0.01,
+        metadata={"help": "Percent of the average Hessian diagonal to use for dampening."},
+    )
+    gptq_block_size: int = field(

```

Comment:
```suggestion
    block_size: int = field(
```

is `grouo_size` defined as follows : `d_model` = `group_size` * `block_size` ?

Reply:
Hi, Ella, The `block_size` is used by DataTrainingArguments for training dataset. So here, we add `gptq` prefix.